CHANGES.rst

```diff
@@ -5,7 +5,8 @@ Changelog
 5.0.4 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Remove len limit of Servizio.ufficio_responsabile .
```

Comment:
Forse meglio qualcosa tipo "Remove selection limit in ufficio_responsabile field for Servizio"?
E il punto senza spazio ;)